config/containers/logging/local.md

```diff
@@ -0,0 +1,70 @@
+---
+description: Describes how to use the local logging driver.
+keywords: local, docker, logging, driver
+redirect_from:
+- /engine/reference/logging/local/
+- /engine/admin/logging/local/
+title: Local File logging driver
+---
+
+
+The `local` logging driver captures output from container's stdout/stderr and
+writes them to a local file on disk. The format of these files is unspecified.
+By default the `local` driver rotates files once they reach 20MB in size. Each
+rotated file is automatically compressed. The default maximum number of files is
+5, for a total of 100MB (when decompressed) of log data.
+
+The goal of this driver is to be as efficient as possible while still
+remaining consistent. As such log files may not be easily parseable with
+standard CLI tools.
+
+## Usage
+
+To use the `local` driver as the default logging driver, set the `log-driver`
+and `log-opt` keys to appropriate values in the `daemon.json` file, which is
+located in `/etc/docker/` on Linux hosts or
+`C:\ProgramData\docker\config\daemon.json` on Windows Server. For more about
+configuring Docker using `daemon.json`, see
+[daemon.json](/engine/reference/commandline/dockerd.md#daemon-configuration-file).
+
+The following example sets the log driver to `local` and sets the `max-size`
+option.
+
+```json
+{
+  "log-driver": "local",
+  "log-opts": {
+    "max-size": "10m"
+  }
+}
+```
+
+Restart Docker for the changes to take effect for newly created containers. Existing containers do not use the new logging configuration.
+
+You can set the logging driver for a specific container by using the
+`--log-driver` flag to `docker container create` or `docker run`:
+
+```bash
+$ docker run \
+      --log-driver local --log-opt max-size=10m \
+      alpine echo hello world
+```
+
+### Options
+
+The `local` logging driver supports the following logging options:
+
+| Option      | Description                                                                                                                                                                                                   | Example  value                           |
+|:------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-----------------------------------------|
+| `max-size`  | The maximum size of the log before it is rolled. A positive integer plus a modifier representing the unit of measure (`k`, `m`, or `g`). Defaults to 20MB.                                          | `--log-opt max-size=10m`                 |
+| `max-file`  | The maximum number of log files that can be present. If rolling the logs creates excess files, the oldest file is removed. **Only effective when `max-size` is also set.** A positive integer. Defaults to 5. | `--log-opt max-file=3`                   |
```

Comment:
I should probably comment this on the implementation PR, but isn't "max file" an implementation detail? Can we abstract storage even more, and just allow people to set _"max amount of logs to keep"_ ?